tfrecords/create_tfrecords_str.py

```diff
@@ -11,7 +11,7 @@
 
 References:
```

Comment:
What's the purpose of this script?

Reply:
this is a reference script to create string-base encoded TFRecord

Reply:
I see `tf.io.serialize_tensor()` being used. Why was it needed?

Reply:
This is a script named as 'create_tfrecords.py' previously. I just renamed it as '_str'. You want me te delete this?

Reply:
Oh got it. Sorry, I got confused. Not let's keep it but also indicate the reason why the script exists.